src/core/hooks/useForm.ts

```diff
@@ -181,7 +181,7 @@ function useForm<T = FormData, TData = void>(
           return result;
         } catch (err: any) {
           setSubmitError(
-            err.message ?? (t("An unexpected error ocurred.") as string),
+            err.message ?? (t("An unexpected error occurred.") as string),
```

Comment:
There is an issue with this pattern : `err.message` is coming from the BE and is not translated. Might want a ticket for that..